File: Cloud77.Service.Abstractions/Entity/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Cloud77.Service.Entity
{
    public class TaskEntity
    {
        public string Email { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public int State { get; set; }
    }
}

File: canteen-service/models/database/task.ts
import { MongoClient, Document } from "mongodb";

const Tasks = "Tasks";

export interface TaskEntity extends Document {
    Email: string,
    Title: string,
    Description: string,
    State: number
}

export async function getTasks(client: MongoClient, database: string, email: string) {
    await client.connect();
    const db = client.db(database);
    const tasks = (await db.collection<TaskEntity>(Tasks).find({ Email: email }).toArray());
    await client.close();
    return tasks;
}
